niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -2159,17 +2212,22 @@ public void onError(int error) {
 
         mPermissionTasks.put(permission.MANAGE_ROLE_HOLDERS,
                 new PermissionTest(false, Build.VERSION_CODES.Q, () -> {
-                    RoleManager roleManager = (RoleManager) mContext.getSystemService(
-                            Context.ROLE_SERVICE);
-                    invokeReflectionCall(roleManager.getClass(), "getRoleHolders", roleManager,
-                            new Class[]{String.class}, RoleManager.ROLE_SMS);
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
Nit: This check shouldn't be necessary since the min SDK version of the task is set to Android Q.

Reply:
it's for avoidng android lint check.  Sdk28(project sdk) does't support RoleManager class.

Reply:
Thanks for confirming! Marking as resolved to avoid the lint warnings / errors.